web/src/components/AuthResults/AuthResults.js

```diff
@@ -13,22 +23,28 @@ export default () => {
     setLastUpdate(new Date().toLocaleTimeString())
   }, [currentUser, userMetadata, isAuthenticated])
 
-  return (
-    <div className="w-full overflow-x-auto overflow-y-auto text-sm max-h-80">
-      <p>Last update {lastUpdate}</p>
-      <code>
-        userMetaData:
-        <pre style={{ margin: 0 }}>{JSON.stringify(userMetadata, null, 2)}</pre>
-      </code>
-      <br />
-      <code>
-        currentUser:
-        <pre style={{ margin: 0 }}>{JSON.stringify(currentUser, null, 2)}</pre>
-      </code>
+  const authData = (userMetadata, currentUser) => {
```

Comment:
I'd move this function out of the component. No need to recreate it on every render.

Reply:
@Tobbe  Thank you,  will implement that once I get out of class and look into highlightjs